src/group_theory/specific_groups/quaternion.lean

```diff
@@ -148,24 +148,17 @@ def quaternion_group_zero_equiv_dihedral_group_zero : quaternion_group 0 ≃* di
   right_inv := by rintro (k | k); refl,
   map_mul' := by { rintros (k | k) (l | l); { dsimp, simp, }, } }
 
-/-- Some of the lemmas on `zmod m` require that `m` is positive, as `m = 2 * n` is the case relevant
-in this file but we don't want to write `[fact (0 < 2 * n)]` we make this lemma a local instance. -/
-private lemma succ_mul_pos_fact {m : ℕ} [hn : fact (0 < n)] : fact (0 < (nat.succ m) * n) :=
-⟨nat.succ_mul_pos m hn.1⟩
-
-local attribute [instance] succ_mul_pos_fact
-
 /--
 If `0 < n`, then `quaternion_group n` is a finite group.
 -/
-instance [fact (0 < n)] : fintype (quaternion_group n) := fintype.of_equiv _ fintype_helper
+instance as [ne_zero n] : fintype (quaternion_group n) := fintype.of_equiv _ fintype_helper
```

Comment:
What is the meaning/purpose of "as" here?

Reply:
a typo, oops!